build/common.sh

```diff
@@ -229,6 +238,9 @@ SHELL=${SHELL} BLOCKSIZE=${BLOCKSIZE} MAIL=${MAIL} PATH=${PATH} \
 TERM=${TERM} HOSTTYPE=${HOSTTYPE} VENDOR=${VENDOR} OSTYPE=${OSTYPE} \
 MACHTYPE=${MACHTYPE} PWD=${PWD} GROUP=${GROUP} HOST=${HOST} \
 EDITOR=${EDITOR} PAGER=${PAGER} ABI_FILE=${ABI_FILE}"
+if [ ${PRODUCT_ARCH} == armv6 ] || [ ${PRODUCT_ARCH} == armv7 ] || [ ${PRODUCT_ARCH} == aarch64 ]; then
+       export ENV_FILTER="${ENV_FILTER} ALLOW_UNSUPPORTED_SYSTEM=yes"
```

Comment:
it's in the make.conf. Why do we need it here?

Reply:
Thanks, I have reverted the commit.